tinyagentos/routes/projects.py

```
@@ -477,6 +479,16 @@ class CreateTaskIn(_TaskRequestModelMixin, BaseModel):
     parent_task_id: str | None = None
     element_id: str | None = None
 
+    @model_validator(mode="after")
+    def _assert_body_for_claimable(self) -> CreateTaskIn:
+        labels = self.labels
+        has_claimable = any(l.strip().lower() == "claimable" for l in labels)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)